### PR TITLE
Adding code of conduct and contribution guidelines 

### PR DESCRIPTION
**Pull request by atbagga**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Search existing github issues for similar questions and feel free to file an issue with a "question" tag to get our attention. 
+
+# Found a Bug
+
+#### Where to Find Known Issues
+
+Before you file a new bug report do check on existing open [issues](https://github.com/Microsoft/vsts-cli/issues?q=is%3Aopen+is%3Aissue+label%3Abug).
+
+If you find your issue already exists, make relevant comments and add your reaction. Use a reaction in place of a "+1" comment.
+👍 - upvote 👎 - downvote
```
Comment (gauravsaralMs) [inline CONTRIBUTING.md]: looks overkill 😄

Comment (geverghe) [reply CONTRIBUTING.md]: This section needs to be retained. We really have to consider the persona of users who are entering the repo with absolutely no context and this is crucial.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+#### Reporting New Issue
+
+The best way to get your bug fixed is to provide good repro steps. Use our [Bug template](https://github.com/Microsoft/vsts-cli/issues/new/choose) to report any issues.
+
+#### Security Bugs
```
Comment (geverghe) [inline CONTRIBUTING.md]: Remove all TO-DO sections if you are targeting the main branch to be available on Day 0. If this is WIP, then its fine.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+TO-DO
+
+# Feature Requests
```
Comment (geverghe) [inline CONTRIBUTING.md]: Sentence casing

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+Before you file a new request do check on existing open [feature requests](https://github.com/Microsoft/vsts-cli/issues?q=is%3Aissue+is%3Aopen+label%3AFeature).
+
+If you find your request already exists, make relevant comments and add your reaction. Use a reaction in place of a "+1" comment.
+👍 - upvote 👎 - downvote
+
+
+#### Submit New Feature request
+
+Use our Feature Request [template](https://github.com/Microsoft/vsts-cli/issues/new/choose) to create a new feature request.
```
Comment (geverghe) [inline CONTRIBUTING.md]: Again, if there is no template knock this line off and just direct the users to create an issue with a feature request label.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+# Contribute Code/Documentation Pull Requests
```
Comment (geverghe) [inline CONTRIBUTING.md]: Didn't quite get the intent with this title

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+## Your First Contribution
+
+Please go through our [code of conduct](./CODE%20OF%20CONDUCT.md) and CLA requirements.
```
Comment (geverghe) [inline CONTRIBUTING.md]: Push the code of conduct within doc as the last section.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+You will need to complete a Contributor License Agreement (CLA). Briefly, this agreement testifies that you are granting us permission to use the submitted change according to the terms of the project's license, and that the work being submitted is under appropriate copyright.
```
Comment (geverghe) [inline CONTRIBUTING.md]: This CLA section seems to be too jarring and potentially too scary for a new user. I am thinking how we can make this potentially more subtle.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+Please submit a Contributor License Agreement (CLA) before submitting a pull request. You may visit https://cla.microsoft.com to sign digitally.
+
+## Getting started
```
Comment (geverghe) [inline CONTRIBUTING.md]: Get started

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+# Have a question
+
+Search existing github [issues](https://github.com/Microsoft/vsts-cli/issues?q=is%3Aopen+is%3Aissue+label%3Aquestion) for similar questions first, and feel free to file an issue with a "question" tag to get our attention.
+
+# Found a bug
+
+#### Where to find known issues
+
+Before you file a new bug report do check on existing open [issues](https://github.com/Microsoft/vsts-cli/issues?q=is%3Aopen+is%3Aissue+label%3Abug).
```
Comment (geverghe) [inline CONTRIBUTING.md]: Recommend reword to "Check open issues before you file a new bug report."

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+
+#### Report new issue
+
+The best way to get your bug fixed is to provide good repro steps. Use our [Bug template](https://github.com/Microsoft/vsts-cli/issues/new/choose) to report any issues.
+
+# Feature requests
+
+#### Where to find existing requests
+
+Before you file a new request do check on existing open [feature requests](https://github.com/Microsoft/vsts-cli/issues?q=is%3Aissue+is%3Aopen+label%3AFeature).
```
Comment (geverghe) [inline CONTRIBUTING.md]: Recommend reword - "Check open feature requests before you file a new request"

**Pull request merged**